### PR TITLE
Visualize

### PR DESCRIPTION
**Pull request by phoenisbuster**

### CODE REVIEW

**Comment by produdez** [comment]
1. Remove pptx file from this PR (add later after present is better i think).
2. Notebook file `visual.ipynb` is not enough. I remember mentioning that all graphs/visuals need to be output into picture files (put them in `/images` or `/output` folder)
3. Insights extracted from those visuals should be included in the notebook also. Ex:
 ```
For the pie chart (90-95 grade is the highest proportion of review score that given to the game. 
manymany extremely love the game gave it maxscore 100. Also, there few people dont really like it)
```
4. Something feels off about the `Top adj chart`
image
    
    - (Personal opinion) Should remove the **Common adjectives** like `full`, `old`, .. to really show the **DIFFERENT** adjectives from the two bracket
    - Could even make 3 graphs instead (common adj, low grade unique adj and high grade unique adj)
    - Also, if you notice, I dont think `fish` and `nintendo` are adjectives

**Comment by phoenisbuster** [comment]
@RedEvilBK Please re-check the adj tokenize and export the images

**Comment by RedEvilBK** [comment]
@produdez review plz

File: sort_by_length.csv [hunk 1]
```diff
@@ -0,0 +1,96 @@
+Score,Review length
```
Comment (produdez) [inline sort_by_length.csv:1]: 1. Put this in `/data`
2. Maybe better name if possible (`review_sorted_by_length`)

File: Visualization_Final.ipynb [hunk 2]
```diff
@@ -0,0 +1,1265 @@
+{
```
Comment (produdez) [inline Visualization_Final.ipynb:1]: Recheck this file please, all of its content is gone

**Review by produdez — CHANGES_REQUESTED** [review]
Also, rename the `model-output.csv` back to `model-output.sample.csv` please. This is because each time we run our prediction pipeline, the result order will varies due to batching/parallelized

**Comment by RedEvilBK** [comment]
- Rename back to animal-crossing.sample.csv
- Put sort_by_length.csv in /data
- Fix  .ipynb file 
@produdez  Review the last visualize_update plz & see if anything wrong?

**Review by produdez — APPROVED** [review]

**Pull request merged**